app/src/test/java/com/example/demouniverload/search/presenter/ParsingSiteMethodTest.java

```diff
@@ -0,0 +1,25 @@
+package com.example.demouniverload.search.presenter;
+
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.junit.Assert;
+import org.junit.Test;
+
+
+
+public class ParsingSiteMethodTest {
```

Comment:
It will be better to rename to:
```suggestion
public class SiteToImageUrlsParserTest {
```
Same for `ParsingSiteMethod` class.